content/synthetics/browser_check.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Browser Check
+kind: documentation
+beta: true
+description: Simulate and monitor user journeys from specific locations.
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/uptime_check"
+  tag: "Documentation"
+  text: "Configure an Uptime Check"
+---
+
+## Overview
+
+Browser checks are scenarios executed by Datadog on your web application at configurable periodic intervals, from multiple locations around the world, on multiple devices. These checks verify not only that your applications are up and responding to requests, but that any conditions defined in your scenarios are met.
+
+## Configuration
+
+### Test details
+
+Define the configuration of our Browser check.
+
+{{< img src="synthetics/browser_check/browser_check_configuration.png" alt="Uptime check make request" responsive="true" style="width:80%;">}}
+
+1. Give your Browser Check a name. 
+2. Enter the Start URL, this is URL on which your Browser Check starts its scenario.
+3. Define on which device to run your check, available devices are: `Laptop Large`, `Tablet`, and `Mobile Small`.
+4. Pick-up locations to run the test from, available locations are:
+    * Frankfurt (Request made from an AWS Datacenter)
+    * Tokyo (Request made from an AWS Datacenter)
+5. Choose a Check frequency between 1 run/ 5 min to 1 run/week:
```

Comment:
Are we not talking about tags on purpose? It might be worth adding a blurb on the ability to select tags that will allow filtering on the main Synthetics page afterwards.